avro/src/main/scala/magnolify/avro/AvroType.scala

```diff
@@ -16,7 +16,7 @@
 
 package magnolify.avro
 
-import java.nio.ByteBuffer
+import java.nio.{ByteBuffer, ByteOrder}
 import java.time._
```

Comment:
Maybe we could import joda as `import org.joda.{time => joda}` so that the Joda APIs could simply refer to `joda.LocalDate`?